src/javascript/JContent/JContent.utils.js

```diff
@@ -168,3 +168,16 @@ export const arrayValue = value => {
 };
 
 export const booleanValue = v => typeof v === 'string' ? v === 'true' : Boolean(v);
+
+export const getCanDisplayItemParams = node => {
+    const folders = ['jnt:contentFolder', 'jnt:folder'];
+    const params = {};
+
+    if (folders.includes(node.primaryNodeType.name)) {
+        params.folderNode = node;
+    } else {
+        params.selectionNode = node;
+    }
```

Comment:
the switch between folderNode / selectionNode does not depend on the type, but rather on the context on which the method is being called. A "locate" action will always try to locate a selection node (expandTree is trying to find for the item a possible parent in the available accordions), and the "open in jcontent" will always try to find a folderNode (just looking for an available accordion for the passed node)

Reply:
See my comment for one issue that still have with that locate functionality.